### PR TITLE
balance : increase minimum pay for everyone.

### PR DESCRIPTION
**Pull request by Huffie56**
# About the pull request

rifleman shouldn't be so poor they
can't even afford to buy a pack of the
cheapest cigarette and a zippo...

# Explain why it's good for the game
everyone in the marines should have enought money to buy atleas a pack of the cheapest cigarette or cigar and a zippo.
# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:
qol: increase minimum paygrade for every marines
/:cl:

### CODE REVIEW

**Comment by realforest2001** [comment]
This isn't really balance altering.

**Review by morrowwolf — CHANGES_REQUESTED** [review]
Change the multiplier for marine money rather than the cost of the things please.

**Comment by Huffie56** [comment]
> Change the multiplier for marine money rather than the cost of the things please.

i can't ? can't we use money to buy black market stuff?
way more safer to just reduce items prices?...

**Review by fira — APPROVED** [review]

File: code/datums/paygrades/factions/uscm/marine.dm [hunk 1]
```diff
@@ -11,23 +11,23 @@
...
+	pay_multiplier = 1.6
 
 /datum/paygrade/marine/e2
 	paygrade = "ME2"
 	name = "Private First Class"
 	prefix = "PFC"
 	rank_pin = /obj/item/clothing/accessory/ranks/marine/e2
 	ranking = 1
-	pay_multiplier = 1 // the default.
+	pay_multiplier = 1.6 // the default.
```
[the remark this answers is not in the document]
Comment (Huffie56) [reply code/datums/paygrades/factions/uscm/marine.dm]: it's been executed.

**Review by morrowwolf — APPROVED** [review]

**Pull request merged**